environment/src/main/java/com/sequenceiq/environment/parameters/validation/validators/parameter/AzureParameterValidator.java

```diff
@@ -156,8 +158,16 @@ private ValidationResult validateEncryptionParameters(ValidationResultBuilder va
                                 + "but that feature is currently disabled. Get 'CDP_CB_AZURE_DISK_SSE_WITH_CMK' enabled for your account to use SSE with CMK.").
                         build();
             }
+            if (encryptionKeyResourceGroupName == null && USE_MULTIPLE.equals(azureParametersDto.getAzureResourceGroupDto().getResourceGroupUsagePattern())) {
+                LOGGER.info("Invalid request, Neither --encryption-key-resource-group-name nor --resource-group-name is present.");
+                return validationResultBuilder.error(
+                        "To use Server Side Encryption for Azure Managed disks with CMK, Either --encryption-key-resource-group-name or " +
```

Comment:
1. Make "Either" lowercase
2. Technically it is valid to specify both options at the same time.